hack/dind-cluster.sh

```diff
@@ -272,16 +302,9 @@ function stop() {
   echo "Stopping dind cluster '${cluster_id}'"
   sudo echo -n
 
-  # Delete additional network interface on the nodes if needed
-  local bridge_num=${BRIDGE_START_NUM}
-  for (( i=1; i<=NODE_COUNT; i++ )); do
-    local bridge="${BRIDGE_PREFIX}${bridge_num}"
-
-    sudo ip link set "${bridge}" down 2> /dev/null || "true"
-    sudo brctl delbr "${bridge}" 2> /dev/null || "true"
-
-    (( bridge_num += 1 ))
-  done
+  # Delete additional bridge if present
+  sudo ip link set "${ADDITIONAL_BRIDGE_NAME}" down 2> /dev/null || "true"
+  sudo brctl delbr "${ADDITIONAL_BRIDGE_NAME}" 2> /dev/null || "true"
```

Comment:
That means if you overrode ADDITIONAL_BRIDGE_NAME when starting the cluster, you have to override it when stopping the cluster as well. And you can't use "start -r" to restart the cluster with a different bridge name... It would really be better if the script could figure out the old bridge name here rather than assuming it's ADDITIONAL_BRIDGE_NAME.